reproducibility_tests/latest_comparable_paths.jl

```diff
@@ -0,0 +1,110 @@
+import Dates
+
+function sorted_dataset_folder(; dir = pwd())
+    matching_paths = String[]
+    for file in readdir(dir)
+        !ispath(joinpath(dir, file)) && continue
+        push!(matching_paths, joinpath(dir, file))
+    end
+    isempty(matching_paths) && return ""
+    # sort by timestamp
+    sorted_paths =
+        sort(matching_paths; by = f -> Dates.unix2datetime(stat(f).mtime))
```

Comment:
```suggestion
        sort(matching_paths; by = f -> stat(f).mtime)
```
I don't think you need to convert to DateTime for this to work